source/comply/audit-log.rst

```diff
@@ -1,369 +1,293 @@
-Configure audit log in Mattermost
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Configuring the Mattermost server to use the new audit log requires editing the ``config.json`` file directly. The audit settings output audit records to allow any combination of local file, syslog, and TCP socket targets.
-
-File target supports rotation and compression triggered by size and/or duration. Syslog target supports local and remote syslog servers, with or without TLS transport. TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+Audit logging provides System Admins, including Security, IT/SRE, Compliance, and HR/PeopleOps teams, with a method of recording activities and events performed within a Mattermost workspace, such as access to the REST API or mmctl. Mattermost audit logging offers you control over where the audit logs are generated and stored, and applies a standard :doc:`JSON schema </comply/embedded-json-audit-log-schema>` to log output.
 
 .. note::
-   Mattermost v6.0 introduced a change to the logging engine library which changes how the logger is configured in Mattermost 5.x versions. Please refer to :download:`Advanced Logging sample file <../samples/sample-logger-config.json>` for a sample configuration file supported from v6.0. 
 
-   
-**Accessing configuration options for audit log**
+  - Logs are recorded asynchronously to reduce latency to the caller, and are stored separately from general logging.
+  - During short spans of inability to write to targets, the audit records buffer in memory with a configurable maximum size cap. Based on typical audit record volumes, it could take many minutes to fill the buffer. After that, the records are dropped, and the record drop event is logged.
+  - When using syslog remotely, we recommend writing to a local file so no records are lost. Records will be taken from the local file and sent to the remote syslog when syslog becomes available again.
+
+Configure audit logging
+-----------------------
 
-Open ``config.json`` and navigate to the audit settings under ``ExperimentalAuditSettings``. Within the setting ``AdvancedLoggingConfig`` you are able to specifiy a filespec to another config file, a database DSN, or JSON. 
+Configuring Mattermost to enable audit logging requires editing the ``config.json`` file directly. Audit logging can’t be managed using the System Console.
 
-The logging configuration JSON is an object (unordered collection) containing names and log target values. Each log target contains a type, options specific to the target type, format, and levels.
+In the ``config.json`` file, go to the ``ExperimentalAuditSettings`` section. Within the ``AdvancedLoggingConfig`` setting, you can specify an absolute or relative filespec to another configuration file, a database DSN, or a JSON string. The process of configuring audit logging includes specifying destination targets, event names to include, and the verbosity of the audit log output.
```

Comment:
User can't set DSN themselves - that is only used when the whole config.json comes from a database.  I think it is enough to document file or escaped JSON string.  Otherwise users my think it can fetch config from any arbitrary database.